dist/js/index.js

```diff
@@ -0,0 +1,9 @@
+import Slider from './slider/slider.js';
+
+window.onload = () => {
+    const slidersArray = Array.from(document.querySelectorAll('.section__content-slider'));
+    slidersArray.forEach((slider) => {
+        const sliderClass = new Slider(slider);
+        sliderClass.addSliderEventListeners();
```

Comment:
I would like to see more common `init` method :)

Reply:
fixed